app/uk/gov/hmrc/exports/models/declaration/AdditionalDeclarationType.scala

```diff
@@ -16,15 +16,21 @@
 
 package uk.gov.hmrc.exports.models.declaration
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json._
 
-case class AdditionalDeclarationType(additionalDeclarationType: String)
-
-object AdditionalDeclarationType {
-  implicit val format: OFormat[AdditionalDeclarationType] = Json.format[AdditionalDeclarationType]
-
-  object AllowedAdditionalDeclarationTypes {
-    val Simplified = "Y"
-    val Standard = "Z"
-  }
+object AdditionalDeclarationType extends Enumeration {
+  type AdditionalDeclarationType = Value
+  implicit val format: Format[AdditionalDeclarationType.Value] =
+    Format(
+      Reads(
+        _.validate[String]
+          .filter(s => AdditionalDeclarationType.values.exists(_.toString == s))
+          .map(s => AdditionalDeclarationType.values.find(_.toString == s).get)
```

Comment:
Hmm, I think that will be better with just `find` without checking `exists` and getOrElse with exception.

The same proposal like on frontend about using `s` and `v`

Reply:
Hmmm, i'd rather we get a JsError if we send something unexpected, it will make it obvious if we've missed something.

I'll change the `s` and `v`